main.js

```diff
@@ -19,15 +19,19 @@ function Hoverable() {
 	var win = window;
 
 	function init() {
-		win.document.body.setAttribute('data-o-hoverable--js', '');
+		win.document.documentElement.setAttribute('data-o-hoverable--js', '');
 		touchSupport();
 	}
 
+	function updateClasses(action) {
```

Comment:
Ok, although I've just ran `ag o-hoverable-if-hover-enabled --sass | grep -i demos` on all of our repos and the only demo that sets the variable is… the module's demo itself.